Julia Entities blog post (#307)
JuliaLang/www.julialang.org: blog/_posts/2019-02-11-julia-entities.md

## Julia Stewards

One official group of people representing the Julia project does exist: [Julia Stewards](https://julialang.org/community/stewards/). This group exists to handle conflict resolution and reports of inappropriate or problematic behavior in the Julia community and ecosystem. The current stewards are: Milan Bouchet-Valat, Simon Byrne, Tim Holy, Katharine Hyatt, Steven Johnson, Stefan Karpinski, and Viral Shah. This group gets (rare) emails about potentially problematic situations and confidentially discusses and decides on what to do about them. The stewards group is not responsible for making any kinds of technical decisions or steering the project in any other way—it just exists to handle complaints and disputes, which fortunately are exceedingly rare.

## The Julia Lab

> A lot of the early work on the development of core Julia was done at MIT in what is now known as [The Julia Lab](https://julia.mit.edu) under the direction of professor [Alan Edelman](https://en.wikipedia.org/wiki/Alan_Edelman). All of the co-creators of Julia have been part of the lab at some point. In those early days, we didn’t call it the Julia Lab yet, it was just an office where some people worked on high performance computing, numerical linear algebra, [random matrix theory](https://en.wikipedia.org/wiki/Random_matrix), and this new programming language called Julia. Although not formally part of the Julia Lab, Dr Jeremy Kepner of Lincoln Laboratory was an early believer in and funder of the lab’s work on Julia and continues to support this work. Since those early days, things have gotten a lot more organized under the leadership (in chronological order) of [Jiahao Chen](https://jiahao.github.io), [Andreas Noack](https://github.com/andreasnoack) and [Valentin Churavy](https://github.com/vchuravy), and the Julia Lab remains a steady source of major innovations and contributions to Julia.

Author: ```suggestion
A lot of the early work on the development of core Julia was done at MIT in what is now known as [The Julia Lab](https://julia.mit.edu) under the direction of professor Alan Edelman. All of the co-creators of Julia have been part of the lab at some point. In those early days, we didn’t call it the Julia Lab yet, it was just an office where some people worked on high performance computing, numerical linear algebra, [random matrix theory](https://en.wikipedia.org/wiki/Random_matrix), and this new programming language called Julia. Although not formally part of the Julia Lab, [Jeremy Kepner](https://www.ll.mit.edu/biographies/jeremy-kepner) of Lincoln Laboratory was an early believer in and funder of the lab’s work on Julia and continues to support this work. Since those early days, things have gotten a lot more organized under the leadership (in chronological order) of [Jiahao Chen](https://jiahao.github.io), [Andreas Noack](https://github.com/andreasnoack) and [Valentin Churavy](https://github.com/vchuravy), and the Julia Lab remains a steady source of major innovations and contributions to Julia.
```

What kind of money goes to Julia through NumFOCUS and how is it spent? When you click on the “[Donate to Julia](https://www.flipcause.com/secure/cause_pdetails/MjI1Nw==)” button, your donation goes to NumFOCUS and is earmarked to be used for the Julia project. This money is used to pay for infrastructure and personnel costs, including website hosting, continuous integration services, and anyone who is working on Julia through NumFOCUS (no one at the moment). By far the biggest flow of money through NumFOCUS for Julia is for the annual Julia conference—JuliaCon. Money from JuliaCon ticket sales and sponsorships go to NumFOCUS which then pays vendors for services required to make JuliaCon happen and reimburse anyone who has reimbursable conference-related expenses. Excess money (if any) from JuliaCon goes into the same pool as donations. JuliaCon finances are managed by the finance chair and approved by the JuliaCon organizing committee as representatives of the project and community. All money that goes in and out of NumFOCUS is a matter of [public record](https://numfocus.org/legal) since it is a nonprofit.

## Julia Computing

> In 2015, the four co-creators of Julia along with [Keno Fischer](https://www.linkedin.com/in/keno-fischer-974261b0/) and [Deepak Vinchhi](https://www.linkedin.com/in/deepakvinchhi/) founded [Julia Computing](https://juliacomputing.com/), a United States C corporation. Julia Computing provides products, support and services around Julia, including [JuliaBox](https://juliabox.com), [JuliaPro](https://juliacomputing.com/products/juliapro.html), [JuliaTeam](https://juliacomputing.com/products/juliateam.html) and [JuliaAcademy](https://academy.juliabox.com). The company employs many of the top contributors to Julia itself (half of the top ten contributors by commits) and to major Julia packages, especially for data science and machine learning. Even though there’s no formal connection between the company and the open source project, Julia Computing is deeply committed to Julia’s success and is proud to sponsor JuliaCon each year and drive so much development of open source Julia code.

Author: ```suggestion
In 2015, the four co-creators of Julia along with [Keno Fischer](https://www.linkedin.com/in/keno-fischer-974261b0/) and [Deepak Vinchhi](https://www.linkedin.com/in/deepakvinchhi/) founded [Julia Computing](https://juliacomputing.com/), a United States C corporation. Julia Computing provides products, support and services around Julia, including [JuliaBox](https://juliabox.com), [JuliaPro](https://juliacomputing.com/products/juliapro.html), [JuliaTeam](https://juliacomputing.com/products/juliateam.html) and [JuliaAcademy](https://academy.juliabox.com). The company employs many of the top contributors to Julia itself (half of the top ten contributors by commits) and to major Julia packages, especially for data science and machine learning. Even though there’s no formal connection between the company and the open source project, Julia Computing is deeply committed to Julia’s success and is proud to sponsor JuliaCon each year and drive so much open source Julia development.
```

## Conclusion

> I hope that this clarifies what the entities surrounding the Julia project are and how they relate to each other. This picture will certainly evlove and change over time, and as it does so we'll try to keep an accurate description of these entities and relationships available as they develop.

Reviewer: "evlove" -> "evolve"

Author: ```suggestion
I hope that this clarifies what the entities surrounding the Julia project are and how they relate to each other. This picture will certainly evolve and change over time, and as it does so we'll try to keep an accurate description of these entities and relationships available as they develop.
```
